src/debugger/channel/FileIOWrapper.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+/* @flow */
+
+import typeof fs from "fs";
+import path from "path";
+import { MessagePackager } from "./MessagePackager.js";
+import invariant from "../../invariant.js";
+
+export class FileIOWrapper {
+  // fs cannot be imported from anything called directly or indirectly
+  // by prepack-standalone so it needs to be passed in here
+  constructor(isAdapter: boolean, fileSystem: fs, inFilePath: string, outFilePath: string) {
```

Comment:
Again, please hide "fs" inside FileIOWrapper. You do not need to pass "fs" from constructor.

Reply:
FileIOWrapper cannot directly import fs even when it itself is being passed from prepack-node. It is still a dependency to prepack-standalone. I tried this yesterday.

Reply:
Changed to import type DebugChannel so fs is hidden inside FileIOWrapper.